docs/quick-start-guide.md

```diff
@@ -65,12 +65,62 @@ CarbonData can be integrated with Spark,Presto and Hive execution engines. The b
 #### Alluxio
 [CarbonData supports read and write with Alluxio](./alluxio-guide.md)
 
+## Installing and Configuring CarbonData to run locally with Spark SQL
+
+To enable CarbonExtensions in spark, we need to add the following configuration.
```

Comment:
One or two points regarding how carbonextensions in spark will help the user can be added here.

Reply:
done